onnxruntime/core/providers/cpu/tensor/reshape.cc

```diff
@@ -32,4 +32,23 @@ ONNX_CPU_OPERATOR_VERSIONED_KERNEL(
         .TypeConstraint("T", DataTypeImpl::AllTensorTypes()),
     Reshape_1);
 
+ONNX_CPU_OPERATOR_VERSIONED_KERNEL(
```

Comment:
there is also a Reshape registration for opset 13 above:
https://github.com/microsoft/onnxruntime/pull/7177/files#diff-ac8e4404b2e6f213f5b99435d1e654a707388711904ce217f547e85cf4c9efccR17
should that one be removed?